common/src/main/kotlin/org/neo4j/connectors/kafka/data/DynamicTypes.kt

```diff
@@ -208,6 +322,18 @@ object DynamicTypes {
   }
 
   fun toConnectSchema(
+      payloadMode: PayloadMode = PayloadMode.EXTENDED,
```

Comment:
is there a need for the default value? I think it'd rather be explicit (even for tests)

Reply:
I think you're right. Since the function behavior depending on this parameter, making it explicit would remove the ambiguity.